Snowflake.Data.Tests/Client/DateTimeOffsetWrapper.cs

```diff
@@ -0,0 +1,9 @@
+using System;
+
+namespace Snowflake.Data.Tests.Client
+{
+    public class DateTimeOffsetWrapper
```

Comment:
what for do we need such wrappers?

Reply:
this class in tests scope only and it it used for tests. We used this class to test how different Snowflake time related types are converted into DateTimeOffset.